pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -366,14 +391,17 @@ private Long configureRefreshDelay(final Long refreshDelay) {
 
   @Option(
     names = {"--host-whitelist"},
-    paramLabel = "<hostname>",
+    paramLabel = "<hostname>[,<hostname>...]... or * or all",
     description =
-        "Comma separated list of hostnames to whitelist for RPC access.  default: ${DEFAULT-VALUE}",
+        "Comma separated list of hostnames to whitelist for RPC access or * or all to accept any host.  default: ${DEFAULT-VALUE}",
     defaultValue = "localhost"
   )
   private final JsonRPCWhitelistHostsProperty hostsWhitelist = new JsonRPCWhitelistHostsProperty();
 
+  /** @deprecated Deprecated in favour of --chain option */
+  @Deprecated
   @Option(
+    hidden = true,
     names = {"--dev-mode"},
     description =
         "set during development to have a custom genesis with specific chain id "
```

Comment:
Similar to `--network-id` we should probably keep this as a fully supported option until we have a way to set a fixed, low difficulty in the genesis config (and support `--network DEV`).